ansible/roles/gitlab/templates/etc/default/gitlab-pages.j2

```diff
@@ -0,0 +1,18 @@
+{# Copyright (C) 2011-2015 GitLab B.V. <https://gitlab.com/>
+ # Copyright (C) 2015-2017 Maciej Delmanowski <drybjed@gmail.com>
```

Comment:
You don't need to include my copyright here. :-)

Reply:
It's a key-value file, he. But, ok, i've set my copyright :D